okhttp/src/jvmTest/resources/okhttp3/internal/publicsuffix/NOTICE

```diff
@@ -1,4 +1,4 @@
-Note that PublicSuffixDatabase.gz is compiled from The Public Suffix List:
+Note that PublicSuffixDatabase is compiled from The Public Suffix List:
```

Comment:
Should .list suffix be present?

Reply:
Still a mix of .list and .dat.

Reply:
Yeah, in that directory it’s `public_suffix_list.dat` which is the exact file we get from the publicsuffix.org. Elsewhere it’s our custom transformation of that file for good performance at runtime.